CONTRIBUTING.md

```diff
@@ -23,7 +23,7 @@ You can create a pull request using the standard `gh pr create` command. Here ar
 
 - **Tests:** Our suite of tests will run automatically on the creation of a pr but you can also run them on your local branch by running `./gradlew check`.
 
-- **Creating a local maven .AAR:** If you want to test your changes locally before creating a PR, you can publish your .AAR locally by running `./gradlew publishReleasePublicationToMavenLocal`. Make sure to add the maven local repo to the application that is importing thumbprint so it is fetched from there and not JitPack. If you are a Thumbtack employee add it to BaseModulePlugin like this:
+- **Creating a local maven .AAR:** If you want to test your changes locally before creating a PR, you can publish your .AAR locally by running `./gradlew publishReleasePublicationToMavenLocal`. To override the published version name with "local", set `com.thumbtack.android.overrideVersionNameWithLocal` to true in `gradle.properties`. Make sure to add the maven local repo to the application that is importing thumbprint so it is fetched from there and not JitPack. If you are a Thumbtack employee add it to BaseModulePlugin like this:
```

Comment:
We can probably take out the Thumbtack-internal instructions now, since those will be out of date and not really relevant in this public repo.